test/unit/modules/app.js

```diff
@@ -6,13 +6,17 @@ var crypto = require('crypto');
 
 var constants = require('../../../helpers/constants');
 
-describe('app', function () {
+// ToDo: Ensure that different app instance is not running - port collision
```

Comment:
Please, explain why they are skipped.

Reply:
I think it's done - there is a port collision of a test and app which requires node.js file